src/Core/Config/AbstractDbaConfigAdapter.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Friendica\Core\Config;
+
+use Friendica\Database\DBA;
+
+abstract class AbstractDbaConfigAdapter
+{
+	public function isConnected()
+	{
+		return DBA::connected();
```

Comment:
This line here is slowing down my system, so that pages load in 3-4 seconds when they otherwise would load in less than a second.